aggregators/1.0/modules/ROOT/pages/aggregators-group-example.adoc

```diff
@@ -0,0 +1,160 @@
+This generates two HTTP requests to aggregate to the group ID `1`.
+. Send the command `http://localhost:8086/test?groupId=2` to aggregate one request to the group ID `2`.
+. Send another command `http://localhost:8086/test?groupId=1` to aggregate a request to group ID `1`.
+. Navigate to the *Console* view to read the logger message:
+
+```
+INFO  2021-06-14 16:34:33,921 [[MuleRuntime].uber.12: [aggregator-group-demo].aggregator-group-demoFlow.CPU_INTENSIVE @2c3478b5] [processor: aggregator-group-demoFlow/processors/1/route/0/processors/0; event: 8beae010-cd47-11eb-8b82-f01898ad2638] org.mule.runtime.core.internal.processor.LoggerMessageProcessor: Doing incremental aggregation
+INFO  2021-06-14 16:34:43,618 [[MuleRuntime].uber.12: [aggregator-group-demo].aggregator-group-demoFlow.CPU_INTENSIVE @2c3478b5] [processor: aggregator-group-demoFlow/processors/1/route/0/processors/0; event: 91b93be0-cd47-11eb-8b82-f01898ad2638] org.mule.runtime.core.internal.processor.LoggerMessageProcessor: Doing incremental aggregation
+INFO  2021-06-14 16:34:47,203 [[MuleRuntime].uber.12: [aggregator-group-demo].aggregator-group-demoFlow.CPU_INTENSIVE @2c3478b5] [processor: aggregator-group-demoFlow/processors/1/route/0/processors/0; event: 93d8e790-cd47-11eb-8b82-f01898ad2638] org.mule.runtime.core.internal.processor.LoggerMessageProcessor: Doing incremental aggregation
+INFO  2021-06-14 16:34:51,380 [[MuleRuntime].uber.12: [aggregator-group-demo].aggregator-group-demoFlow.CPU_INTENSIVE @2c3478b5] [processor: aggregator-group-demoFlow/processors/1/route/1/processors/0; event: 96566ab0-cd47-11eb-8b82-f01898ad2638] org.mule.runtime.core.internal.processor.LoggerMessageProcessor: Aggregation complete
+INFO  2021-06-14 16:34:51,409 [[MuleRuntime].uber.12: [aggregator-group-demo].aggregator-group-demoFlow.CPU_INTENSIVE @2c3478b5] [processor: aggregator-group-demoFlow/processors/1/route/1/processors/1; event: 96566ab0-cd47-11eb-8b82-f01898ad2638] org.mule.runtime.core.internal.processor.LoggerMessageProcessor: [
+  {
+    "Source Name": "group-based",
+    "GroupId": "1",
+    "Source ID": "0.5127046416902801"
+  },
+  {
+    "Source Name": "group-based",
+    "GroupId": "1",
+    "Source ID": "0.20239307122656847"
+  },
+  {
+    "Source Name": "group-based",
+    "GroupId": "1",
+    "Source ID": "0.7505233647798346"
+  }
+]
+```
+This example logger message shows that for the first two HTTP requests with group ID `1` and the third request with group ID `2`, the *Logger* component in the *Incremental aggregator* route logs the message `Doing incremental aggregation step.` +
+After you sent the fourth HTTP request with group ID `1`, the aggregation group `1` is complete because of the group size of `3` calls per group. The first *Logger* component in the *Aggregation complete* route logs the message `Aggregation complete`, and the second *Logger* component returns the aggregated payload elements of the group, which are requests transformed into random ID numbers.
```

Comment:
```suggestion
After you send the fourth HTTP request, which is the third request with group ID `1`, the aggregation group `1` is complete because of the group size of `3` calls per group. The first *Logger* component in the *Aggregation complete* route logs the message `Aggregation complete`, and the second *Logger* component returns the aggregated payload elements of the group, which are requests transformed into random ID numbers.
```